app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -71,14 +70,18 @@ class AttendeeFragment : Fragment() {
         })
 
         rootView.register.setOnClickListener {
-            val attendee = Attendee(id = attendeeFragmentViewModel.getId(),
-                    firstname = firstName.text.toString(),
-                    lastname = lastName.text.toString(),
-                    email = email.text.toString(),
-                    ticket = ticketId,
-                    event = eventId)
-
-            attendeeFragmentViewModel.createAttendee(attendee)
+            ticket.map {
+                if (it.second > 0) {
+                    val attendee = Attendee(id = attendeeFragmentViewModel.getId(),
+                            firstname = firstName.text.toString(),
+                            lastname = lastName.text.toString(),
+                            email = email.text.toString(),
+                            ticket = TicketId(it.first.toLong()),
+                            event = eventId)
+
+                    attendeeFragmentViewModel.createAttendee(attendee)
+                }
+            }
```

Comment:
No use of map here

Reply:
need to create multiple attendees depending upon list of Pair(ticketId,TicketQty)

Reply:
So, use forEach, not map